server/statistics/region_collection.go

```diff
@@ -232,6 +245,8 @@ func (r *RegionStatistics) Collect() {
 	regionStatusGauge.WithLabelValues("pending-peer-region-count").Set(float64(len(r.stats[PendingPeer])))
 	regionStatusGauge.WithLabelValues("learner-peer-region-count").Set(float64(len(r.stats[LearnerPeer])))
 	regionStatusGauge.WithLabelValues("empty-region-count").Set(float64(len(r.stats[EmptyRegion])))
+	regionStatusGauge.WithLabelValues("over-size-region-count").Set(float64(len(r.stats[OversizeRegion])))
+	regionStatusGauge.WithLabelValues("under-size-region-count").Set(float64(len(r.stats[UndersizeRegion])))
```

Comment:
It seems all regions will be divided into these two types. So maybe we can use another monitor to record these normal situations.